docs/docs/linking-to-a-route.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Linking to a route
+sidebar_label: Linking to a route
+---
+
+Now that we have created our router, let's create our first link to our route:
+
+```tsx {8,9} title="src/Header.tsx"
+import { Link } from "@swan-io/chicane/Link";
+import { Router } from "./router";
+
+const Header = () => {
+  return (
+    <div>
+      <h1>My super app</h1>
+      <Link to={Router.Home()}>Home</Link>
+      <Link to={Router.Users()}>Users</Link>
+    </div>
+  );
+};
+```
+
+The `Link` component takes a few props:
+
+- `to`: (**required**): The route you're linking to
+- `className`
+- `activeClassName`
+- `style`
+- `activeStyle`
+- `replace`: replace instead of push
+
+## Creating your own Link component
+
+We provide a default `Link` component, but you can also create yours if needed using the `useLink` hook:
+
+```tsx {4} title="src/Link.tsx"
+import { useLink } from "@swan-io/chicane/useLink";
+
+const Link = ({ className, activeClassName, to, ...props }) => {
+  const { active, onClick } = useLink({ href: to, replace });
+
+  return (
+    <a
+      {...props}
+      href={to}
+      onClick={onClick}
+      className={
```

Comment:
No need to add the whole active link style for the useLinkProps example imho, as it will probably be [baseStyle, active && activeStyle] or something like that in real usages